### PR TITLE
PIL-8 bFix/add trackid header

### PR DESCRIPTION
**Pull request by alazarevski**
Add trackid (lowercase) header param extracted from the offer.trackId on takeOffer call

### CODE REVIEW

File: src/models/Offer.js [hunk 1]
```diff
@@ -54,6 +55,7 @@ export type OfferRequest = {
   provider: string,
   fromAssetAddress: string,
   toAssetAddress: string,
+  trackId: string,
```
Comment (poocart) [inline src/models/Offer.js]: Do we have it for fiat offers as well? 🤔

Comment (alazarevski) [reply src/models/Offer.js]: I don't think so. The changes show it as in Fiat, but the param is added to OfferRequest.

File: src/services/exchange.js [hunk 2]
```diff
@@ -143,7 +143,7 @@ export default class ExchangeService {
     return axios.post(
       buildApiUrl('orders', '2.0'),
       JSON.stringify(order),
```
Comment (poocart) [inline src/services/exchange.js]: Remove `trackId` from payload here?

File: src/models/Offer.js (hunk 1 above)
Comment (poocart) [reply src/models/Offer.js]: Yeah, sorry, mislooked! All good here 👍

File: src/services/exchange.js (hunk 2 above)
Comment (alazarevski) [reply src/services/exchange.js]: 06824ea

**Pull request merged**